wsi/physicaldisplay.h

```diff
@@ -63,6 +63,10 @@ class PhysicalDisplay : public NativeDisplay, public DisplayPlaneHandler {
   bool SetActiveConfig(uint32_t config) override;
   bool GetActiveConfig(uint32_t *config) override;
 
+  bool SetResolution(const HwcRect<int32_t>&) override;
+  bool IsCustomResolution();
+  const HwcRect<int32_t> &GetResolution();
```

Comment:
GetCustomResolution or GetFloatingRect ? same for setter ?

Reply:
I can change to GetCustomResolution and same for set.